composer.json

```
@@ -27,7 +27,7 @@
     }
   , "require": {
         "php": ">=5.4.2"
-      , "ratchet/rfc6455": "^0.3.1"
+      , "ratchet/rfc6455": "^0.3.1 | ^0.4.0"
```

nit: Can you amend your commit to list "preferred" versions first? (refs #1105 and others)


```suggestion
      , "ratchet/rfc6455": "^0.4 || ^0.3.1"
```